odoo/addons/base/models/res_partner.py

```diff
@@ -61,7 +61,7 @@ def _fields_view_get_address(self, arch):
                 #(e.g fields not present on the model). In that case we just return arch
                 if self._name != 'res.partner':
                     try:
-                        self.env['ir.ui.view'].postprocess_and_fields(self._name, sub_view_node, None)
+                        self.env['ir.ui.view'].postprocess_and_fields(self._name, sub_view_node, None, validate=False)
```

Comment:
Beware: here the purpose is to validate the view!

Reply:
I'm not sure, validate is similar to check_field_names context key that wasn't given here before.